src/hooks/UrqlUseSubscriptionWithHandler.re

```diff
@@ -1,38 +1,42 @@
+type handler('a, 'b, 'c) =
+  | Handler((option('a), 'b) => 'a): handler('a, 'b, 'a)
+  | NoHandler: handler(_, 'b, 'b);
+
 [@bs.deriving abstract]
-type useSubscriptionWithHandlerArgs = {
+type useSubscriptionArgs = {
   query: string,
   [@bs.optional]
   variables: Js.Json.t,
 };
 
 [@bs.deriving abstract]
-type useSubscriptionWithHandlerResponseJs('acc) = {
+type useSubscriptionResponseJs('a) = {
```

Comment:
I'd prefer to stick with `'acc` if you're ok with that. I'm more and more convinced that it's nice to have our type parameter names be meaningful for future readers of the code.

Reply:
Sure!